numpy/lib/_npyio_impl.py

```diff
@@ -1574,8 +1577,7 @@ def first_write(self, v):
 
         # Handle 1-dimensional arrays
         if X.ndim == 0 or X.ndim > 2:
-            raise ValueError(
-                "Expected 1D or 2D array, got %dD array instead" % X.ndim)
+            raise ValueError("Expected 1D or 2D array, got %dD array instead" % X.ndim)
```

Comment:
Please keep this line as before, it now exceeds 79 characters (also see https://numpy.org/doc/stable/dev/index.html#stylistic-guidelines)

Reply:
I will revert the changes and keep it as it was before.